gto-support-scarlet/src/main/java/com/tinder/scarlet/internal/utils/RuntimePlatformInternals.java

```diff
@@ -0,0 +1,12 @@
+package com.tinder.scarlet.internal.utils;
+
+import androidx.annotation.NonNull;
+
+class RuntimePlatformInternals {
```

Comment:
This is a package private Java class to hide it from the public API for library consumers